docs/source/installation/migration/1-migrating-project.rst

```diff
@@ -125,28 +124,40 @@ will be unable to submit forms or sync with the server.
   * ``./manage.py dump_domain_data <domain_name>`` 
   * ``./manage.py run_blob_export --all <domain_name>``
 
-* Transfer these two files to the new environment.
+  .. note::
+     It is important to have the commit hash that ``dump_domain_data`` and ``run_blob_export`` were run from. If
+     Dimagi does not provide you with this commit hash, please followup to ensure you are able to reference this
+     hash in future steps.
+
+* Transfer these files to the new environment.
 
 .. note::
-  If you are not able to use your own domain for a test run and would like dump data for a test domain for practising or testing, please contact us via https://forum.dimagi.com/c/developers/ with subject "Request for test domain dump data for migration testing" and mention this page. A dimagi developer will provide you above data for any test/QA domains (casesearch, ccqa, dataregistry, qateam, ben-test, qa-erm-v1-downstream1) from https://staging.commcarehq.org.
+  If you are not able to use your own domain for a test run and would like dump data for a test domain for
+  practising or testing, please contact us via https://forum.dimagi.com/c/developers/ with subject
+  "Request for test domain dump data for migration testing" and mention this page. A dimagi developer will
+  provide you above data for any test/QA domains (casesearch, ccqa, dataregistry, qateam, ben-test, qa-erm-v1-downstream1)
```

Comment:
nit: missing "the"

Reply:
oh whoops I didn't even see that you merged this.

Reply:
You're in luck cause I have another PR open 😁 updated with some additional changes here 43d064a688ba563e24e58813e3f713414a33d3f1